### PR TITLE
refactor: big changes of dashboard

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What problem does this PR solve?

First, sorry this PR is relatively huge, I'm a little busy last past days, so I didn't split it into several little PRs.

This PR aims to give the dashboard an overall optimization.

### What is changed and how does it work?

- A concise interface
- Now can create an experiment from YAML file
- Make label_selectors as a HTML select element
- Now can search events

  > Also, you can search by some useful patterns, for example:
  > **namespace:default network-delay-exmaple** will search events with namespace default.
  > **kind:NetworkChaos network-delay-exmaple** will search events with kind NetworkChaos, you can also type **kind:net**, it will also work because the search is fuzzy.
  > **time:2020-06-30 network-delay-exmaple** will search events before 2020-06-30, the time pattern only supports year, month, and day.

- Click timeline legends will zoom to the last event position by experiment name
- Init archives page
- update .ico files to adapt dark mode

### CODE REVIEW

File: ui/src/pages/Archives/index.tsx [hunk 1]
```diff
@@ -1,6 +1,83 @@
...
+  }
+
+  return (
+    <ContentContainer>
+      <Grid container spacing={3}>
+        {archives &&
+          archives.length > 0 &&
+          archives.map((a) => (
+            <Grid key={a.Name} item xs={12}>
+              <ExperimentPaper experiment={a} isArchive handleSelect={setSelected} handleDialogOpen={setDialogOpen} />
```
Comment (yeya24) [inline ui/src/pages/Archives/index.tsx:56]: Do we need to specify `isArchive={true}` here? If not specified, I thought it will use the default value false

**Comment by yeya24** [comment]
Could you please sign the DCO?

And after I deployed one experiment, the frontend crashed. Can you take a look?
Screenshot from 2020-06-30 14-05-05

File: ui/src/components/StatusBar/index.tsx [hunk 2]
```diff
@@ -64,10 +64,6 @@ export const CurrentStatus: React.FC<CurrentStatusProps> = ({ classes, state })
       label: 'Paused',
       value: state.paused,
     },
-    {
-      label: 'Finished',
-      value: state.finished,
-    },
```
[the remark this answers is not in the document]
Comment (g1eny0ung) [reply ui/src/components/StatusBar/index.tsx:70]: This is a, how to say, A tricky place during my development, I found everyone has its own opinion to display these statuses.

Now, I think I need to make a setting page and let users customize the statuses they want to display.

[the remark this answers is not in the document]
Comment (yeya24) [reply ui/src/components/StatusBar/index.tsx:70]: It's totally okay to use the current one as well. I don't want to add more work to you.

File: ui/src/pages/Archives/index.tsx (hunk 1 above)
Comment (g1eny0ung) [reply ui/src/pages/Archives/index.tsx:56]: No need, if the boolean prop not explicitly specified, It will be considered as true.

**Comment by g1eny0ung** [comment]
> Could you please sign the DCO?
> 
> And after I deployed one experiment, the frontend crashed. Can you take a look?
> Screenshot from 2020-06-30 14-05-05

After debugging, I found this should be the case where I did not handle the experimental error correctly. I will fix it soon.

**Comment by g1eny0ung** [comment]
@yeya24 The crash problem should be fixed.

And I try to rebase git to add the sign-off, but the hard is I'm already merging the upstream update twice, so it's complicated to add it to every single commit pushed by me. 😵

What about adding the sign-off before this PR ready to merge to master? This can save a lot of time. I will remember to sign it in the later PRs.

**Comment by cwen0** [comment]
image

**Comment by cwen0** [comment]
Can we add a delete button on the label selector? 
image

**Comment by g1eny0ung** [comment]
@cwen0 The legends problem should be fixed and also, now you can easily delete the labels.

**Comment by g1eny0ung** [comment]
@yeya24 Thx.~ 😆

Generally speaking, this is usually handled in the front-end. I will think about how to implement it. At present, the feature of the **search experiment** has not been completed. I can put the **sorting** into the search and complete it together.

**Pull request merged**